terraform/ipfs-server/process-stream.sh

```diff
@@ -9,12 +9,9 @@ M3U8_SIZE=10
 function startFFmpeg() {
   while true; do
     mv /var/log/ffmpeg /var/log/ffmpeg.1
-    offset=0
-    if [ -f ~/stream-offset ]; then
-      offset=$(cat ~/stream-offset)
-    fi
-    ffmpeg -nostats -re -i "${RTMP_STREAM}" -f mpegts -vcodec copy -output_ts_offset ${offset} -hls_time ${HLS_TIME} -hls_list_size 10 -f hls ${what}.m3u8 > /var/log/ffmpeg 2>&1
-    sleep 1
+    echo 1 > ~/streamReset
+    ffmpeg -nostats -re -i "${RTMP_STREAM}" -f mpegts -vcodec copy -hls_time ${HLS_TIME} -hls_list_size 10 -f hls ${what}.m3u8 > /var/log/ffmpeg 2>&1
+    sleep 0.2
```

Comment:
This is really aggressive eh, if ffmpeg keeps failing for whatever reason. Maybe 0.5?

Reply:
The lower the value the faster the stream will pick up again.  I dont even really need it but i figured a bit of a buffer not a bad idea?